include/ListView/ListViewSmarty.php

```diff
@@ -133,12 +133,15 @@ public function process($file, $data, $htmlpublic)
 
         $totalWidth = 0;
         foreach ((array)$this->displayColumns as $name => $params) {
-            $totalWidth += (int)$params['width'];
+            $totalWidth += (isset($params['width']))? (int)$params['width'] : 0;
```

Comment:
Unnecessary parenthesis can be removed:

```suggestion
            $totalWidth += isset($params['width']) ? (int)$params['width'] : 0;
```

Reply:
Fixed in https://github.com/salesagility/SuiteCRM/pull/7251/commits/502f9726b7cad005c7b0e2439d26273228484cfa